sample_structure.py

```diff
@@ -27,7 +27,7 @@ class StructureSampler():
     everything in braces are the respective indices.
 
     '''
-    def __init__(self, args, adsorbate_index=None, bulk_indices_list=None):
+    def __init__(self, args):
```

Comment:
We should record all the different options supported by sample_structure in a readme.md script. Let's add this as a separate PR.